source/nordic_sdk/components/ble/peer_manager/id_manager.h

```diff
@@ -229,6 +229,31 @@ ret_code_t im_wlist_create(pm_peer_id_t        * p_peer_ids,
  */
 bool im_address_resolve(ble_gap_addr_t const * p_addr, ble_gap_irk_t const * p_irk);
 
+/**@brief Function for calculating the ah() hash function described in Bluetooth core specification

```

Comment:
@LiyouZhou I have just added the `@note` as you requested. See https://github.com/ARMmbed/nrf51-sdk/pull/17/files#diff-999146604efce71bca279e64873ab799R255.